testsuite/MDAnalysisTests/test_transformations.py

```diff
@@ -17,8 +17,8 @@
 from six.moves import range
 
 import numpy as np
-from numpy.testing import assert_allclose, assert_equal, TestCase
-import math
+from numpy.testing import assert_allclose, assert_equal
```

Comment:
+1 for removing `TestCase`. I don't understand why we actually need it, even under `nose`, since a test class inheriting from `object` will work fine, even with fixtures like `setUp` and such.